src/services/fence.js

```diff
@@ -133,12 +133,21 @@ export const deleteFenceTokens = async (api, fence) => {
  * Download File From a Fence Data Repository
  */
 export const downloadFileFromFence = async ({ fileUuid, api, fence }) => {
-  let accessToken = await getAccessToken(api, fence);
+  let accessToken = null;
+  try {
+    accessToken = await getAccessToken(api, fence);
+  } catch (err) {
+    console.warn(`[Fence] no access token fo file ${fileUuid}, assuming open access`);
```

Comment:
Typo 'fo' -> 'to'
Should we mentionned the file uuid in the error message, because access token is not related to the file?